src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.ts

```diff
@@ -205,12 +236,20 @@ export class WorkItemDetailComponent implements OnInit {
   }
 
   save(): void {
```

Comment:
I would recommend to separate save and create method for mainly two reason - 

- Better readability.
- In very near future we are going to support partial update, when this will become more complex.

actually let's discuss about it.